README.md

```diff
@@ -90,6 +90,22 @@ You are now ready to start a local development server on http://localhost:3000 v
 npm run dev
 ```
 
+## Development
+
+### TypeScript
+
+Since this app uses TypeScript and the data ist stored in a PostgreSQL database, we need a way to ensure type-safety for the requested data. We do this by fetching the types from the database schema and converting them to TypeScript types, to be found in `./src/lib/types/database.ts`.
+
+Whenever we need to update the types because of a database schema change, we can so by running `npm run generate-types`.
+
+Note that there are 3 requirements for running the script:
+
+- the [Supabase CLI](https://supabase.com/docs/guides/cli) installed locally as it does the actual type generation
+- Docker (required for the Supabase command)
+- Setting `export DATABASE_URL="postgresql://user:password@your-postgrest-server.com/your-table"` (or any other way of managing your shell variables)
```

Comment:
Maybe better:

```
- Making environment variable `DATABASE_URL="postgresql://<USER>:<PASSWORD>@<HOST>/<DATABASE>"` available for the script 
```
Since `export=...` makes the assumption that we are in a POSIX compatible shell